src/main/scala/com/fulcrumgenomics/umi/DuplexConsensusCaller.scala

```diff
@@ -238,8 +283,8 @@ class DuplexConsensusCaller(override val readNamePrefix: String,
         // Call the duplex reads
         val duplexR1Sources = filteredAbR1s ++ filteredBaR2s
         val duplexR2Sources = filteredAbR2s ++ filteredBaR1s
-        val duplexR1 = duplexConsensus(abR1Consensus, baR2Consensus, duplexR1Sources)
-        val duplexR2 = duplexConsensus(abR2Consensus, baR1Consensus, duplexR2Sources)
+        val duplexR1 = duplexConsensus(abR1Consensus, baR2Consensus, duplexR1Sources.toArray[SourceRead])
```

Comment:
The method being called still takes `Seq[SourceRead`, so this is confusing.  I think either the the signature on `duplexConsensus` needs to change, or this should remain a `Seq`, and if you want arrays for performance, you can do that internally to `duplexConsensus`.

Reply:
Done